src/04-modality-specific-files/05-task-events.md

```diff
@@ -45,9 +47,8 @@ the first volume.
 
 An arbitrary number of additional columns can be added. Those allow describing
 other properties of events that could be later referred in modelling and
-hypothesis extensions of BIDS.
-Note that the `trial_type` and any additional columns in a TSV file SHOULD be
-documented in an accompanying JSON sidecar file.
+hypothesis extensions of BIDS. Note that the `trial_type` and any additional
+columns in a TSV file SHOULD be documented in an accompanying JSON sidecar file.
```

Comment:
we try to enforce some "soft" criterion that sentences should be started on a new line always.

And if a sentence grows too long, it needs to be broken over several lines, preferably at meaningful spots semantically.

So I guess you wanted to save space (yay!), but squeezing sentences into single lines makes the git diff harder to read in the future (nooo!).

Reply:
Gotcha. Will revert that change.

Not even sure when I changed that.

Reply:
@sappelhoff Fixed.
Also applied the rule to some paragraphs earlier in the doc. Hope I did it the "right way".